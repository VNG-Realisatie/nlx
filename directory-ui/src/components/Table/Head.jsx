// Copyright Â© VNG Realisatie 2020
// Licensed under the EUPL
//

import { node } from 'prop-types'
import styled from 'styled-components'

const Head = styled.thead`
  display: table-header-group;
`

Head.propTypes = {
  children: node,
}

export default Head
