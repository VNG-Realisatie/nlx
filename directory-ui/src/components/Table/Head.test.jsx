import React from 'react'
import { shallow } from 'enzyme'
import Head from './Head'

it('should match snapshot', () => {
  expect(shallow(<Head/>)).toMatchSnapshot()
})

it('should render child elements', () => {
  expect(shallow(<Head>
    <tr>
      <th>Table head</th>
    </tr>
  </Head>).contains(<tr>
    <th>Table head</th>
  </tr>)).toEqual(true)
})