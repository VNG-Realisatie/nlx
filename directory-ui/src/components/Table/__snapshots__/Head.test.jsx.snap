// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    display: table-header-group;
",
        ],
      },
      "displayName": "styled.thead",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "children": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "thead",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
